src/contracts.js

```diff
@@ -23,6 +23,15 @@ const formData = [
       multiselect: false,
     },
   },
+  {
+    title: "ContractType",
+    referenceValues: {
+      source: "contractType",
+      reference: "contracts/contract-types",
+      optionText: "value",
+      multiselect: false,
+    },
+  },
```

Comment:
contract type should be next to company: 
![form](https://user-images.githubusercontent.com/12630904/214854742-8dc108e5-72ed-4053-9661-c8b6a6b14bc4.png)